### ISSUE TITLE
`ovh_order_cart` doesn't support the US subsidiary

### ISSUE THREAD

**Issue by zimbatm** [ovh/terraform-provider-ovh#217]
### Terraform Version

```
$ terraform -v
Terraform v1.0.5
on linux_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v2.23.0
+ provider registry.terraform.io/hashicorp/aws v3.43.0
+ provider registry.terraform.io/hashicorp/local v2.1.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/nixpkgs/secret v1.1.1
+ provider registry.terraform.io/ovh/ovh v0.15.0
```

### Affected Resource(s)

- `data.ovh_order_cart`
- `ovh_ip_service`
- `ovh_vrack`
- ???

### Terraform Configuration Files

```hcl
provider "ovh" {
  endpoint = "ovh-us"
}

data "ovh_order_cart" "mycart" {
 ovh_subsidiary = "us"
 description    = "my cart"
}
```

### Debug Output
I don't think it's needed for this issue.

### Panic Output
No panic

### Expected Behavior

Support the "us" value.

### Actual Behavior

```
╷
│ Error: Value us is not among valid values ([cz de es eu fi fr gb ie it lt ma nl pl pt sn tn])
│ 
│   with data.ovh_order_cart.mycart,
│   on vrack.tf line 2, in data "ovh_order_cart" "mycart":
│    2:  ovh_subsidiary = "us"
│ 
╵
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
Not really. We are using the US cloud that's all.

### References
Nothing found on the issue tracker so far.

**Comment by zimbatm** [comment]
Workaround:

Go to https://api.us.ovhcloud.com/console/#/order/cart#POST and create a cart manually there, capture the cardId from the response body.

**Comment by zimbatm** [comment]
Updated the issue with more affected resources

**Comment by zimbatm** [comment]
Fixed by #224
